### ISSUE TITLE
go to OuiShare Fest 2016

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#314]
May 18-20, 2016

#### Schedule

 - Thursday, 15h15 - 16h00, [Measuring and Distributing Value](http://ouisharefestparis2016.sched.org/event/6eSa/measuring-contributions-distributing-value) panel
 - Friday, 15h00 - 15h45, meet with Part-up (https://github.com/gratipay/inside.gratipay.com/issues/314#issuecomment-217968908)

**Comment by whit537** [comment]
2015: #114

**Comment by whit537** [comment]
Registration is open: http://2016.ouisharefest.com/join#_tickets.

> The second piece of good news is that we just opened our **Call for Contributions** and invite you to submit your proposals for the Fest program untill **31 December**! We look forward to your applications.

**Comment by whit537** [comment]
https://ouisharefest.typeform.com/to/TTSxdV

> All accepted contributors receive free admission per proposal to all 3 days of OuiShare Fest, **startup pitches excluded**.

**Comment by whit537** [comment]
> All people who submitted a complete proposal to this form are guaranteed an early bird ticket.

**Comment by whit537** [comment]
I want to talk about take-what-you-want salaries. I think that's the most interesting thing we've done so far.

**Comment by whit537** [comment]
Looks like as long as we submit a proposal before December 31, we are guaranteed the early-bird pricing. Let's sit on this for a bit. I think that Platform Cooperativism (#384) in a couple weeks will give us a better sense of how Gratipay would do at OuiShare, so let's get through that before taking further action here. Sound right?

**Comment by whit537** [comment]
@jesswhit Wanna go to Paris with me in May? :)

**Comment by techtonik** [comment]
> I want to talk about take-what-you-want salaries. I think that's the most interesting thing we've done so far.

I think that most people are also forced to use take-what-you-need, and for that, to estimate the needed capacity of the team I also find a @rummik dashboard pretty exciting - https://pub.rummik.com/expenses/

![image](https://cloud.githubusercontent.com/assets/515889/11060006/7357dcd6-87ae-11e5-83c7-e4ee2a57ca45.png)

We should add this to /~user/expenses

**Comment by whit537** [comment]
@techtonik Yeah, there were a couple folks doing that. I think @patcon had an implementation as well, iirc.

**Comment by whit537** [comment]
Based on my experience at Platform Coops, I think a talk/conversation about FOSS governance models and the collaborative economy could be interesting. Something like, "Does benevolent dictatorship have a role in the collaborative economy?" could be provocative and lead to fruitful conversation.

**Comment by whit537** [comment]
To: partners@ouisharefest.com
Subject: partnership opportunities?

> Greetings!
>
I represent a startup called [Gratipay](https://gratipay.com/), which offers "payments and payroll for open work." Our roots are in the open source community, and we've identified the collaborative economy community as another in which we would like to participate, which we started doing in earnest [at the recent Platform Cooperativism conference](https://medium.com/gratipay-blog/a-platform-cooperative-almost-94cd5900840f).
>
I'm writing to ask, what partnership opportunities are available for OuiShareFest 2016? :-)
>
Thanks!
>
P.S. Please note that, because Gratipay is an open company, I'll need to at least summarize our email conversation publicly for the Gratipay community. That'll happen on [this GitHub thread](https://github.com/gratipay/inside.gratipay.com/issues/314).

**Comment by whit537** [comment]
> Something like, "Does benevolent dictatorship have a role in the collaborative economy?" could be provocative and lead to fruitful conversation.

This could fit with the "After the Gold Rush" theme, depending on how they're interpreting that.

**Comment by whit537** [comment]
Or here's one that's right under our nose: "Openness and Consent".

**Comment by whit537** [comment]
> 5 years on and the enchantment with the “Sharing Economy” is over. Many who set out on the initial journey did not make it, and for now, only very few found gold.

http://magazine.ouishare.net/2015/12/after-the-gold-rush

**Comment by whit537** [comment]
Heard back from OuiShare folks. We're scheduling a call for some time in January.

**Comment by techtonik** [comment]
Awesome theme. The most useful thing that I found useful from startup movements are post-mortems. Definitely need to get there this year. I'd like to talk about psychology, about demands and limitations of us as a humans.

**Comment by whit537** [comment]
Call re: partnership is January 12 at 3pm UTC. Still determining venue ...

@techtonik Well, we've got plenty of post-mortem analysis to offer. :) Do you think we should offer a session? How should we focus it? Should we focus on a specific theme (legal, openness vs. safety), or make it more general?

**Comment by whit537** [comment]
Maybe a "Failing Up" session? "Bring a story of failure to share with the rest of us. We'll have a conversation about the failures we've experienced in our collaborative economy projects and hopefully we can learn from each other's mistakes." Seems like the "workshop" format would make sense?

![screen shot 2015-12-30 at 9 31 19 am](https://cloud.githubusercontent.com/assets/134455/12051968/3a69cdf2-aed8-11e5-9163-d738eab2eab5.png)

**Comment by whit537** [comment]
(Though I suppose we may as well pitch Gratipay in a separate proposal.)

**Comment by whit537** [comment]
> (Though I suppose we may as well pitch Gratipay in a separate proposal.)

Or not. I hate pitches.

**Comment by whit537** [comment]
I just bought two tickets(!).

**Comment by whit537** [comment]
Alright, can we get a proposal in?

**Comment by whit537** [comment]
Okay! Form submitted. Key points:

> **Tell us about yourself in a few words!**
>>I'm a social entrepreneur with roots in the Python open-source community. I'm best known for founding Gratipay, a crowdfunding and payroll service for commons-related projects.

-------------

> **What is one main question you are seeking to address with your proposal?**
*Be original and specific! Questions such as "how the collaborative economy helps address social issues" might be too vague, while "Is Uber part of the collaborative economy" has already been extensively discussed. A good question has a specific angle and combines at least two themes (work and tech for example, or politics and environment).*
>>What unexamined assumptions hinder the growth of the collaborative economy?

------------

>**Describe your expertise on the subject you want to address with your contribution!**
*If you have published papers, given talks,  presentations or are leading a project in this area, this is the right time to tell us and share related links!*
>
>> I've been working on Gratipay for almost four years, during which time we've processed over US$1M. We've gone through several major crises as a company, so I've learned a lot of things the hard way about pursuing a collaborative economy project.
>>
Here's a couple short (~5m) videos showing my general presentation style:
https://vimeo.com/59291936
https://vimeo.com/72813386#t=45m37s
>>
Here's a longer video of me in a workshop setting:
https://www.youtube.com/watch?v=BqSczmUp35w

--------------

> **Workshop abstract**
*Please describe the workshop in less than 1200 characters*
> 
>> When might centralization be the right choice for a collaborative economy project? Hierarchy? Dictatorship? Secrecy? Coercion? Let's spend some time surfacing and examining our assumptions and taboos together, in order to contribute to a more nuanced self-understanding for the collaborative economy movement.
>
> **Workshop methodology**
*Please describe how you would proceed to organise the workshop, as well as the tools you will use.*
>
>> In this facilitated group conversation, we will spend one-quarter of our time introducing ourselves and sharing experiences related to the topic, one-half in exploration of the themes that emerge, and the final quarter in synthesis and convergence.
>
> **What will be the workshop's deliverables?**
*An object, a prototype, a report, a summary, recommendations, etc.*
>
>>The group will be invited to collaborate online following the session (perhaps following the conference) on a document summarizing our conversation and offering recommendations to the wider collaborative economy community.

**Comment by whit537** [comment]
> Thank you for your application Chad Whitacre!
>
Now sit back and wait while we review your proposal! 
>
Your proposal may be immediately chosen by our group of experts or the community will decide. **You will receive a reply by January 15th latest.**
>
And remember: by having submitted this form, **you are guaranteed an early bird ticket.**

**Comment by whit537** [comment]
Their email and [this](https://drive.google.com/file/d/0B2XLJoIHF74KNTZDRFdvOXdrQTA/view) sez January 25th.

**Comment by whit537** [comment]
Booked plane tickets last night!

**Comment by whit537** [comment]
Call w/ OuiShare about partnership opportunities:

https://www.youtube.com/watch?v=dcaMkwFhyV0.

**Comment by whit537** [comment]
Good call! :dancer:

**Comment by whit537** [comment]
tl;dr:

 - They're planning to announce in February a "OuiShare Fest Forward" program that may be interesting to us to apply for.
 - Francesca is going to see if Gratipay fits on a panel even if the session we proposed isn't accepted.
 - I will plan to punch pennies! :-)

**Comment by whit537** [comment]
> Due to the high number of proposals, we won't be able to feature the workshop as you proposed, however, as I mentioned during our call, we would like you to join a group session about open source / open company projects. Therefore I wanted to let you know that I or a program curator will get back to you once we have more details on how you could get involved in this. 

------------

> Sounds good, Francesca! Keep me posted! :-)

**Comment by techtonik** [comment]
Maybe OuiShare is not the best place to discuss all this stuff after all. Maybe it is a place where people expect to be expired, not hear about yet another failure? =) So maybe they are not ready yet to face the reality, because they don't see a way to change it? I will be interested to gather with people who can think about it and calculate the stuff like doing economic hackaton - not just talk about it.

**Comment by whit537** [comment]
> They're planning to announce in February a "OuiShare Fest Forward" program that may be interesting to us to apply for.

Here's [the announcement](http://us5.campaign-archive2.com/?u=744876426414b018c030e670a&id=383d8a1c67).

http://2016.ouisharefest.com/forward

**Comment by whit537** [comment]
Looks like Fest Forward happens during the conference. I think I'd rather just participate in the conference itself.

**Comment by whit537** [comment]
I've booked a place to stay during the conference.

**Comment by whit537** [comment]
> I hope all is well for you these days. 
I wanted to reach out to you regarding your stage involvement at the OuiShare Fest in May. We're so happy to have you around! 
>
We'd love to have you speak at a session called (provisionally) a session called "Tools for Measuring Contributions & Distributing Value" (provisional title), where you'd be very welcome to talk about Gratipay. 
>
It will be on Day 2 (19 May), most probably in the afternoon although there will be some future changes to the timing. 
>
The idea is to introduce the most innovative models we know for managing the difficult question of value distribution. It will be a panel with different organizations - here are two of those we have in mind: 
>
[redacted since I'm not sure they're confirmed yet]
>
How does that sound to you? 
>
I'm also cc-ing [] who will give you all the information needed on the logistics part.

---------------

> This sounds wonderful! Thank you so much for the invitation! [response re: other organizations]
>
Keep up the great work with planning the conference! Let me know what else you need from me. :-)
>
Merci beaucoup! :D

**Comment by whit537** [comment]
> P.S. As a heads-up: we are (publicly) tracking our OuiShare Fest preparations here:
>
https://github.com/gratipay/inside.gratipay.com/issues/314
>
You'll see that I've copied your message there (_without_ your names or the names of the other organizations you're inviting for this session, for privacy).

**Comment by whit537** [comment]
> Great you will be joining us. I have noticed that you are still missing a photo and a biography on your sched profile. Please upload this as soon as possible so I can register you as a speaker and you can appear on our website. 
>
Let me know if you have any questions regarding this,

**Comment by whit537** [comment]
> Thanks, []! I accidentally ended up with two profiles on Sched.org (I didn't know at first that you had created one for me!). I've contacted Sched to delete the second account. The correct one to use has the username **whit537** and is linked to this email address, **chad@zetaweb.com**. Does that give you what you need? :)

**Comment by whit537** [comment]
> I just merged the profiles now and see that all is uploaded, so we're good! :)
>
Thanks!

---------------

> Hmm ... are you sure it worked? I tried going through a "merge" workflow on Sched as well and it didn't seem to work. I believe I am still seeing two accounts:
>
http://ouisharefestparis2016.sched.org/chadwhitacre (bad)
http://ouisharefestparis2016.sched.org/whit537 (good)
>
And on http://ouisharefestparis2016.sched.org/directory/attendees/2:
>
![screen shot 2016-04-07 at 11 24 37 am](https://cloud.githubusercontent.com/assets/134455/14356639/b55bcfaa-fcb3-11e5-81c1-727a51a2fa3c.png)

**Comment by whit537** [comment]
> Check again, I think the first time it may have not worked...I also registered you as a speaker now so you appear on the page :)
Let me know if its still showing up differently for you!

--------------

> Perfect! All set now, thanks []! :D

------------

> Actually, one final thing: should I be listed on the "[Measuring Contributions and Distributing Value](http://ouisharefestparis2016.sched.org/event/6eSa/measuring-contributions-distributing-value)" panel? With that, I could start promoting over Gratipay's Twitter.

**Comment by whit537** [comment]
> Hey, 
It seems that it's not quite confirmed yet which panel you will be in, so perhaps hold off on that for a moment ;) But great that you are helping us promote, i'll let you know once its sure and give you the go! :)

---------

> Ah, okay. Thanks for the heads-up. Keep me posted! :-)

**Comment by whit537** [comment]
> If I could put in a word ... we do seem to be [focusing our storytelling on take-what-you-want compensation](https://github.com/gratipay/inside.gratipay.com/issues/552), and so it did seem like the "distributing value" panel was a really good fit. However, I'm happy to participate wherever you think I can provide the most value to the conference. :-)

**Comment by whit537** [comment]
Okay!

> We'll be "Measuring Contributions & Distributing Value" http://ouisharefestparis2016.sched.org/event/6eSa/measuring-contributions-distributing-value w/ @enspiral's http://cobudget.co/ at @OuiShareFest! 😄 🎉

https://twitter.com/Gratipay/status/719534426858725377

**Comment by whit537** [comment]
> Just finished a call re: http://ouisharefestparis2016.sched.org/event/6eSa/ . Gonna be a great conversation at @ OuiShareFest w/ @ CocoonPro @ Thanksandshare @ cobudget!

https://twitter.com/Gratipay/status/725734703999885312

**Comment by whit537** [comment]
> I hope everything's going well on the other side of the ocean!
I put you in touch with [], from our team, who's in charge of the 3-day accelerator for responsible and open source startups that will take place during the Fest. It's a really cool project! She wanted to ask you if you would accept to mentor some of the project holders. She will tell you more about it.

-----------------

Thanks for thinking of me, []! Great to meet you, []. :-)

I would like to help as a mentor, my only concern is missing out on the rest of the Fest! What does the commitment look like?

**Comment by whit537** [comment]
Commitment is a 30-minute conversation with one of [the participants](http://2016.ouisharefest.com/forward). I've asked for a 45-minute conversation. :-)

> The idea is just to have a 30min talk with Ralph and the rest of the [Part-Up](http://www.part-up.com/) team about value distribution on their platform. Part-Up is a platform for matching temporary teams to work together on projects. They are structured as a cooperative and they want to talk about governance, value distribution and blockchain during their acceleration. You can see more on [this video](https://youtu.be/TKQTzdHjfIY) (2min) they made for the Fest.

**Comment by whit537** [comment]
Okay, we're on for 15h00 to 15h45 on Friday.

**Comment by whit537** [comment]
Panel video: https://www.youtube.com/watch?v=JgynSbU0uOY

https://twitter.com/whit537/status/738110497590018048

**Comment by whit537** [comment]
I'm drafting a report at https://public.etherpad-mozilla.org/p/P8dZpTYnC1. I plan to post a version here once it's settled. I'd like to get a blog post out as well but I'm not sure when that'll happen.

**Comment by whit537** [comment]
I decided the night before OuiShare Fest that my main goal for the conference was to make friends. Friendship is an end in itself, a good for its own sake. I want to spend my life working together on awesome stuff with lots of friends. That's bigger than Gratipay. I came away from the Fest with many new and strengthened relationships:

- I got to meet the OuiShare organizers in person that I had been coordinating with: Francesca, Taoufik, Marguerite, Bianca, and Manuella.
- Francesca led Mourad, Derek, Claudia, and me in a panel discussion on Thursday.
- As a Fest Forward mentor, I learned about Part-up from founders Ralph and Erik, and shared some of Gratipay's experience with them.
- Xavier Damman from Open Collective (#458) and I found ourselves caught in a trust-building exercise together(!). This led to some good conversation, which continued in Lille (#464).
- I participated in a workshop co-led by Tom from Shareable (cf. #403).
- There I met Cynthia from Open Food Network Norway, with whom @JessWhit and I reconnected at the Friday night party along with the ShareTribe founders, Antti and Juho ([cf](https://github.com/gratipay/gratipay.com/commit/02e83a256bceb9edafeab2cba6ab0f00ceb8115d)), and Ossa from Sweden.
- I talked shop with Philippe from ftopia, Louis-David (cf. #637) from Without Model (they published a [book](http://www.withoutmodel.com/without-model/open-models-les-business-models-leconomie-ouverte/) that included sections on Gittip and the Open Company Initiative!), and fellow "consultant in the gift" Jean-François Nourad.
- We danced with Denise from Istanbul, and Nancy from Platform Cooperativism.
- We shared dinner with Platform Cooperativism co-organizer Nathan and his family.
- We met and visited with Richard and Nati and Derek and Chelsea and Halley and Daniel and Craig and others related to Enspiral. Picking up from https://github.com/gratipay/inside.gratipay.com/issues/421#issuecomment-185012209, Richard and I were able to have a solid conversation together in the burrito line—before meeting on the dance floor later that night. :) Derek and I also had some good conversations.
- I told Mara about GitHub Ambridge and she was like, "That's pretty bad-ass." 💃
- Samir taught me TCP.
- I talked and punched pennies with Guillaume, Etienne, Philippe, Anna, Theo, Flora, and hundreds of others.
- I met @simonsarazin from Catalyst, who was to be our host in Lille after the Fest (#464), as well as Elaine, another member of the informal Catalyst community; she works for ANIS, a non-profit providing fiscal sponsorship for Catalyst projects.
- We went to the MangoPay party with Julia and Fran.
- I met Maxime from Inventaire.io—[a Gratipay Team](https://gratipay.com/inventaire-io/)!

After OuiShare Fest we went to Lille for #464, where the relationship-building continued. The format of the event on Monday was a panel discussion with Genevieve from Enspiral/Cobudget, Xavier, me, @Changaco, and Lionel Dricot (aka [Ploum](http://ploum.net/)). @simonsarazin and Marion were our gracious hosts. We stayed in their apartment for two nights along with Changaco, Genevieve, and Carole, who together with Sybille facilitated a "free money" workshop the day after our own panel. We explored Lille and shared meals together with them. I met Guillaume, from whom @simonsarazin originally heard about Gittip back in the day (Guillaume heard about it from [Raphael Hertzog](https://gratipay.com/~rhertzog/)), along with other members of the Catalyst community in Lille.

Phew! Surely there are others I've forgotten. We met a lot of people! :-)

**Comment by whit537** [comment]
![ci5nohpxaaakvb1](https://cloud.githubusercontent.com/assets/134455/15762972/cd29abc2-28ee-11e6-8e1d-2244df9c998b.jpg)

https://twitter.com/aiamax/status/733637909228687361

**Comment by whit537** [comment]
[photos](https://www.flickr.com/photos/ouishare/sets/72157668756824111)

[speaker portraits](https://www.flickr.com/photos/ouishare/sets/72157669148158665/)

https://www.flickr.com/photos/ouishare/27425678111/ ...

![27425678111_86eaab230e_o](https://cloud.githubusercontent.com/assets/134455/15935876/ed6eb060-2e36-11e6-9f91-cc44ea8f4c33.jpg)

**Comment by whit537** [comment]
[all videos](https://www.youtube.com/playlist?list=PL8Bt3EbdmpKOKPy57wesfC7SlWXBjYW6c)

**Comment by whit537** [comment]
Alright! Any reason not to close this? :-)
